diff --git a/node_modules/react-native-confetti-cannon/src/index.js b/node_modules/react-native-confetti-cannon/src/index.js
index 7092e70..cecfecc 100644
--- a/node_modules/react-native-confetti-cannon/src/index.js
+++ b/node_modules/react-native-confetti-cannon/src/index.js
@@ -110,7 +110,7 @@ class Explosion extends React.PureComponent<Props, State> {
 
     const newItems = Array(difference).fill().map((): Item => ({
       leftDelta: randomValue(0, 1),
-      topDelta: randomValue(TOP_MIN, 1),
+      topDelta: randomValue(0.25, 0.75),
       swingDelta: randomValue(0.2, 1),
       speedDelta: {
         rotateX: randomValue(0.3, 1),
@@ -210,8 +210,8 @@ class Explosion extends React.PureComponent<Props, State> {
             outputRange: [0, -(item.swingDelta * 30), (item.swingDelta * 30), 0]
           });
           const opacity = this.animation.interpolate({
-            inputRange: [0, 1, 1.8, 2],
-            outputRange: [1, 1, 1, fadeOut ? 0 : 1]
+            inputRange: [0, 0.1, 1, 1.8, 2], //make confetti not initially visible
+            outputRange: [0, 1, 1, 1, fadeOut ? 0 : 1]
           });
           const containerTransform = [{translateX: left}, {translateY: top}];
           const transform = [{rotateX}, {rotateY}, {rotate: rotateZ}, {translateX}];
