diff --git a/node_modules/react-native-reanimated/lib/module/reanimated2/UpdateProps.js b/node_modules/react-native-reanimated/lib/module/reanimated2/UpdateProps.js
index e69c581..da4ebfd 100644
--- a/node_modules/react-native-reanimated/lib/module/reanimated2/UpdateProps.js
+++ b/node_modules/react-native-reanimated/lib/module/reanimated2/UpdateProps.js
@@ -83,11 +83,13 @@ const createUpdatePropsManager = isFabric() ? () => {
 };
 if (shouldBeUseWeb()) {
   const maybeThrowError = () => {
+    // https://github.com/react-navigation/react-navigation/issues/11729
+    // this error gets thrown in web after upgrading to expo 51. Commenting this out does not seem to break anything though.
     // Jest attempts to access a property of this object to check if it is a Jest mock
     // so we can't throw an error in the getter.
-    if (!isJest()) {
-      throw new Error('[Reanimated] `UpdatePropsManager` is not available on non-native platform.');
-    }
+    // if (!isJest()) {
+    //   throw new Error('[Reanimated] `UpdatePropsManager` is not available on non-native platform.');
+    // }
   };
   global.UpdatePropsManager = new Proxy({}, {
     get: maybeThrowError,
diff --git a/node_modules/react-native-reanimated/lib/module/reanimated2/layoutReanimation/sharedTransitions/ProgressTransitionManager.js b/node_modules/react-native-reanimated/lib/module/reanimated2/layoutReanimation/sharedTransitions/ProgressTransitionManager.js
index cf80db3..75a652c 100644
--- a/node_modules/react-native-reanimated/lib/module/reanimated2/layoutReanimation/sharedTransitions/ProgressTransitionManager.js
+++ b/node_modules/react-native-reanimated/lib/module/reanimated2/layoutReanimation/sharedTransitions/ProgressTransitionManager.js
@@ -193,11 +193,12 @@ function createProgressTransitionRegister() {
 }
 if (shouldBeUseWeb()) {
   const maybeThrowError = () => {
-    // Jest attempts to access a property of this object to check if it is a Jest mock
+    // https://github.com/react-navigation/react-navigation/issues/11729
+    // this error gets thrown in web after upgrading to expo 51. Commenting this out does not seem to break anything though.// Jest attempts to access a property of this object to check if it is a Jest mock
     // so we can't throw an error in the getter.
-    if (!isJest()) {
-      throw new Error('[Reanimated] `ProgressTransitionRegister` is not available on non-native platform.');
-    }
+    // if (!isJest()) {
+    //   throw new Error('[Reanimated] `ProgressTransitionRegister` is not available on non-native platform.');
+    // }
   };
   global.ProgressTransitionRegister = new Proxy({}, {
     get: maybeThrowError,
